### PR TITLE
Add method for validators to check it supports for a given field.

### PR DESCRIPTION
**Pull request by Thisara-Welmilla**
Improvement:
Check whether validators are supported for given field

Related PR:
- https://github.com/wso2/identity-api-server/pull/415

### CODE REVIEW

**Comment by kayathiri4** [comment]
Does JSRegexValidator supports both password and username?

File: components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/Validator.java [hunk 1]
```diff
@@ -43,6 +43,14 @@ public interface Validator {
      */
     boolean validate(ValidationContext context) throws InputValidationMgtClientException;
 
+    /**
+     * Validate whether validator supports given field.
+     *
+     * @param field     Field that validator is needed to be configured.
+     * @return  boolean
+     */
+    boolean isAllowedField(String field);
```
Comment (DMHP) [inline components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/Validator.java:52]: Add a default value.

Comment (Thisara-Welmilla) [reply components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/Validator.java]: Addressed with commit bb6657099a140a701ae31461ce2aaf4626301835

**Review by Thumimku — APPROVED** [review]
LGTM

**Pull request merged**